promiseitwonthurt.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+const workshopper = require('workshopper')
+    , path        = require('path')
+
+function fpath (f) {
+  return path.join(__dirname, f)
```

Comment:
Could. I just used the same `fpath` decl you'll find in most workshoppers, for consistency.